### PR TITLE
Konstantinos / Update Financial Commission certificate of membership - ROW and EU 

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

Update Financial Commission certificate of membership 

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by DerivFE** [comment]
| Name | Result |
| :--- | :------ |
| **Build status**  | Building 🔨 |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6771006612) |

**Comment by DerivFE** [comment]
**Preview Link**: https://membership-changes.deriv-com-preview-links.pages.dev
| Name | Result |
| :--- | :------ |
| **Build status**  | Completed ✅ |
| **Preview URL**  | [Visit Preview](https://membership-changes.deriv-com-preview-links.pages.dev) |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6771006612) |

File: gatsby-node.js
Comment (mitra-deriv) [inline gatsby-node.js:1]: You need to add this for different languages as well like [line 224](https://github.com/binary-com/deriv-com/pull/5860/files#diff-a6f3904c41211be170d9b0eb87e49ea8d02d0fd5a7224d6772174b425dacf90fL224)

**Comment by DerivFE** [comment]
| Name | Result |
| :--- | :------ |
| **Build status**  | Building 🔨 |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6781508604) |

**Comment by DerivFE** [comment]
**Preview Link**: https://membership-changes.deriv-com-preview-links.pages.dev
| Name | Result |
| :--- | :------ |
| **Build status**  | Completed ✅ |
| **Preview URL**  | [Visit Preview](https://membership-changes.deriv-com-preview-links.pages.dev) |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6781508604) |

**Comment by DerivFE** [comment]
| Name | Result |
| :--- | :------ |
| **Build status**  | Building 🔨 |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6781831096) |

**Comment by DerivFE** [comment]
**Preview Link**: https://membership-changes.deriv-com-preview-links.pages.dev
| Name | Result |
| :--- | :------ |
| **Build status**  | Completed ✅ |
| **Preview URL**  | [Visit Preview](https://membership-changes.deriv-com-preview-links.pages.dev) |
| **Action URL**  | [Visit Action](https://github.com/binary-com/deriv-com/actions/runs/6781831096) |

**Review by shayan-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**